lib/miq_automation_engine/service_models/miq_ae_service_physical_server.rb

```diff
@@ -0,0 +1,11 @@
+module MiqAeMethodService
+  class MiqAeServicePhysicalServer < MiqAeServiceModelBase
+    expose :ext_management_system, :association => true
+    expose :physical_servers,      :association => true
+
+    espose :turn_on_loc_led
+    espose :turn_off_loc_led
+    espose :power_on
+    espose :power_off
+  end
+end
```

Comment:
Please add a new-line at the end of this file.  It will resolve the "Final newline missing." rubocop error.